src/Tasks/Resources/Strings.resx

```diff
@@ -221,6 +221,9 @@
 
         If this bucket overflows, pls. contact 'vsppbdev'.
   -->
+  <data name="Copy.LinklibraryFailedPrefix">
+    <value>The link() library call failed with the following error code: </value>
```

Comment:
I don't think the user needs to know what function was called internally. I'd expect something like
``` 
Failed to link '{0}' to '{1}'. Error code {2}.
```
Or the same without the paths.

Reply:
I seem to recall this is part of a longer error message (definitely not confusing), which would mean they get that as well.

I don't feel strongly about mentioning link(); my argument in favor of this version is that it tells people not to look for the subsequent error code in MSBuild. I don't think that's a super strong argument.

Reply:
The log should be similar with https://github.com/dotnet/msbuild/pull/8211#issue-1472875383.  The bold text will be replaced with the current localized string. Sorry, I don't have a non-windows system on my hand now.

"CopyFiles:
Creating hard link to copy "Program.cs" to "c:\MyProject\Destination\Program.cs".
Could not use a hard link to copy "Program.cs" to "c:\MyProject\Destination\Program.cs". Copying the file with symbolic link instead. The system cannot move the file to a different disk drive. (Exception from HRESULT: 0
x80070011)
Creating symbolic link to copy "Program.cs" to "c:\MyProject\Destination\Program.cs".
Could not use a link to copy "Program.cs" to "c:\MyProject\Destination\Program.cs". Copying the file instead. **A required privilege is not held by the client. (Exception from HRESULT: 0x80070522)**
Copying file from "E:\Test\ConsoleApp2\ConsoleApp2\Program.cs" to "c:\MyProject\Destination\Program.cs".
Done Building Project "E:\Test\ConsoleApp2\ConsoleApp2\ConsoleApp2.csproj" (default targets)."

Reply:
FYI: https://github.com/dotnet/msbuild/pull/8213#discussion_r1061545472
(if we'd mention the API, we should mention it properly)

Reply:
done